wordpress-mobile/gutenberg-mobile
# Add VideoPress translation domain

Related PR: https://github.com/Automattic/jetpack/pull/29894.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5622.

**To test:**

1. Run the command `npm run i18n:update` to download translations.
2. Change the device language to a non-English one.
3. Open the app and connect it to the local Metro server.
4. Add VideoPress block.
5. Try the different user flows and observe that texts are translated into the selected language.

**NOTE:** Current translations can be found in `src/i18n-cache/jetpack-videopress-pkg/data`. New ones (i.e. only used in the native version) are added to the localization strings files so they can be requested along with new strings from main apps (https://github.com/wordpress-mobile/gutenberg-mobile/pull/5631/commits/b1bb141f1d1bebb91852ea2765bef9c90170c892). Note that until these get translated their associated texts will be displayed in English.

PR submission checklist:
